FlorenceBML/BMLacq679/BMLacq679.xml

```diff
@@ -35,6 +36,135 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 5</idno>
                         </altIdentifier>
                     </msIdentifier>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <watermark/>
```

Comment:
empty    `<watermark/>` should better be deleted where not applicable

Reply:
Done.